app/src/main/java/io/chatcamp/app/database/ChatCampDatabaseHelper.java

```diff
@@ -0,0 +1,111 @@
+package io.chatcamp.app.database;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.provider.BaseColumns;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.chatcamp.app.ConversationMessage;
+import io.chatcamp.sdk.Message;
+
+import static io.chatcamp.app.database.ChatCampDatabaseContract.DATABASE_NAME;
+import static io.chatcamp.app.database.ChatCampDatabaseContract.DATABASE_VERSION;
+import static io.chatcamp.app.database.ChatCampDatabaseContract.MessageEntry;
+
+/**
+ * Created by shubhamdhabhai on 15/03/18.
+ */
+
+public class ChatCampDatabaseHelper extends SQLiteOpenHelper {
+
+    public ChatCampDatabaseHelper(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase sqLiteDatabase) {
+        String CREATE_MESSAGE_TABLE = "CREATE TABLE IF NOT EXISTS " + MessageEntry.TABLE_NAME + "("
+                + MessageEntry._ID + " INTEGER PRIMARY KEY," + MessageEntry.COLUMN_NAME_GROUP_ID + " TEXT,"
+                + MessageEntry.COLUMN_NAME_MESSAGE + " TEXT," + MessageEntry.COLUMN_NAME_TIME_STAMP + " INTEGER" + ")";
```

Comment:
Are we storing channel type also in DB? If not, then please do so. It is required.